scraper/src/Scraper.hs

```diff
@@ -12,11 +15,15 @@ runScrape = do
   case userOption of
     "1" -> putStrLn "Do the default flow"
     "2" -> putStrLn "Do the everything in database flow"
-    _   -> putStrLn "Oh god what happened"
+    _   -> putStrLn "Oh god what happened" --do the default flow anyway...?
```

Comment:
This sounds reasonable. One thing you'll want to do at a later date is use something like `optparse-generic` or `optparse-applicative` to get better command line parsing behavior.